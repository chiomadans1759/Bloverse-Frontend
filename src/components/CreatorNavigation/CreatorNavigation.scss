#bl-creator-navigation {
  .nav-link {
    font-size: 1rem;
    color: #565656 !important;
    border: none !important;
    height: 2.15625rem;
    background-color: transparent;
  }
  
  .nav-link.active {
    border-bottom: 0.0625rem solid #004ded !important;
  }

  .navigation-bar {
    overflow-x: scroll;
  }

  .nav-tabs {
    flex-wrap: nowrap !important;
  }

  .profile-img {
    border-radius: 50% !important;
  }
}
