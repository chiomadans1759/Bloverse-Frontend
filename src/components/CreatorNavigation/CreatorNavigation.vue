<template>
  <main id="bl-creator-navigation">
    <div class="creator-info mx-2 mb-4 d-block d-sm-none">
      <div class="row">
        <div class="col-3 text-center pr-0">
          <img src="@/assets/Avatar2.png" class="profile-img rounded" alt="...">
        </div>
        <div class="col-9 pl-0 d-flex align-items-center">
          <div class="user-info">
            <h6 class="font-weight-bold">JohnDoe</h6>
            <p>John Doe</p>
            <p><i class="fal fa-map-marker-alt"></i><span class="ml-1">Lagos, Nigeria</span></p>
          </div>
        </div>
      </div>
    </div>
    <div class="navigation-bar">
      <ul class="nav nav-tabs mb-3" id="pills-tab" role="tablist">
        <li class="nav-item d-sm-none">
          <router-link
            to="/creators/info"
            class="nav-link d-flex align-items-center"
            id="pills-contest-tab"
            data-toggle="pill"
            role="tab"
            aria-controls="pills-contest"
            aria-selected="true"
          >Info
          </router-link>
        </li>
        <li class="nav-item">
          <router-link
            to="/creators/profile"
            class="nav-link active d-flex align-items-center"
            id="pills-general-tab"
            data-toggle="pill"
            role="tab"
            aria-controls="pills-general"
            aria-selected="false"
          >Dashboard</router-link>
        </li>
        <li class="nav-item">
          <router-link
            to="/creators/posts"
            class="nav-link d-flex align-items-center"
            id="pills-contest-tab"
            data-toggle="pill"
            href="#posts-tab"
            role="tab"
            aria-controls="pills-contest"
            aria-selected="true"
          >Posts
            <span class="badge badge-secondary ml-2">4</span>
          </router-link>
        </li>
        <li class="nav-item">
          <router-link
            to="/creators/drafts"
            class="nav-link d-flex align-items-center"
            id="pills-contest-tab"
            data-toggle="pill"
            href="#photocontest-tab"
            role="tab"
            aria-controls="pills-contest"
            aria-selected="false"
          >Drafts
            <span class="badge badge-secondary ml-2">4</span>
          </router-link>
        </li>
        <li class="nav-item">
          <router-link
            to="/creators/trash"
            class="nav-link d-flex align-items-center"
            id="pills-contest-tab"
            data-toggle="pill"
            href="#photocontest-tab"
            role="tab"
            aria-controls="pills-contest"
            aria-selected="false"
          >Trash</router-link>
        </li>
      </ul>
    </div>
  </main>
</template>

<script>
export default {
  
}
</script>

<style lang="scss" scoped>
@import "./CreatorNavigation";
</style>
