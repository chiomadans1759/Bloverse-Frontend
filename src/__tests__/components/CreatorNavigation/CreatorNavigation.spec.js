import { mount } from '@vue/test-utils'
import CreatorNavigation from '@/components/CreatorNavigation/CreatorNavigation.vue';

describe('CreatorNavigation Component', () => {
  it('should render correctly', () => {

    const wrapper = mount(CreatorNavigation , {});
    expect(wrapper.element).toMatchSnapshot();
  });
});
