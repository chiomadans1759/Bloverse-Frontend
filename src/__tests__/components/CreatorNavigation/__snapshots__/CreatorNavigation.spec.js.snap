// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreatorNavigation Component should render correctly 1`] = `
<main
  id="bl-creator-navigation"
>
  <div
    class="creator-info mx-2 mb-4 d-block d-sm-none"
  >
    <div
      class="row"
    >
      <div
        class="col-3 text-center pr-0"
      >
        <img
          alt="..."
          class="profile-img rounded"
          src="@/assets/Avatar2.png"
        />
      </div>
       
      <div
        class="col-9 pl-0 d-flex align-items-center"
      >
        <div
          class="user-info"
        >
          <h6
            class="font-weight-bold"
          >
            JohnDoe
          </h6>
           
          <p>
            John Doe
          </p>
           
          <p>
            <i
              class="fal fa-map-marker-alt"
            />
            <span
              class="ml-1"
            >
              Lagos, Nigeria
            </span>
          </p>
        </div>
      </div>
    </div>
  </div>
   
  <div
    class="navigation-bar"
  >
    <ul
      class="nav nav-tabs mb-3"
      id="pills-tab"
      role="tablist"
    >
      <li
        class="nav-item d-sm-none"
      >
        <a
          aria-controls="pills-contest"
          aria-selected="true"
          class="nav-link d-flex align-items-center"
          data-toggle="pill"
          id="pills-contest-tab"
          role="tab"
        >
          Info
        
        </a>
      </li>
       
      <li
        class="nav-item"
      >
        <a
          aria-controls="pills-general"
          aria-selected="false"
          class="nav-link active d-flex align-items-center"
          data-toggle="pill"
          id="pills-general-tab"
          role="tab"
        >
          Dashboard
        </a>
      </li>
       
      <li
        class="nav-item"
      >
        <a
          aria-controls="pills-contest"
          aria-selected="true"
          class="nav-link d-flex align-items-center"
          data-toggle="pill"
          href="#posts-tab"
          id="pills-contest-tab"
          role="tab"
        >
          Posts
          
          <span
            class="badge badge-secondary ml-2"
          >
            4
          </span>
        </a>
      </li>
       
      <li
        class="nav-item"
      >
        <a
          aria-controls="pills-contest"
          aria-selected="false"
          class="nav-link d-flex align-items-center"
          data-toggle="pill"
          href="#photocontest-tab"
          id="pills-contest-tab"
          role="tab"
        >
          Drafts
          
          <span
            class="badge badge-secondary ml-2"
          >
            4
          </span>
        </a>
      </li>
       
      <li
        class="nav-item"
      >
        <a
          aria-controls="pills-contest"
          aria-selected="false"
          class="nav-link d-flex align-items-center"
          data-toggle="pill"
          href="#photocontest-tab"
          id="pills-contest-tab"
          role="tab"
        >
          Trash
        </a>
      </li>
    </ul>
  </div>
</main>
`;
